### PR TITLE
B28 g31 93 alexander

### PR DESCRIPTION
**Pull request by Try2Teachme**
Guys please check my code

### CODE REVIEW

**Comment by Try2Teachme** [comment]
I dont know what to do with that conflict

**Review by untuzina — APPROVED** [review]

**Comment by Micovs** [comment]
The conflict should be resolved now.

**Review by Alexa3462 — COMMENTED** [review]
Alexander, do you have any implementations inside of your step definitions? I was not able to see it.

File: src/test/java/com/cloud/runners/CukesRunner.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
         features = "src/test/resources/features",
         glue = "com/cloud/step_definitions",
         dryRun = false,
-        tags = "@alev",
+        //tags = "@alev",
```
Comment (Alexa3462) [inline src/test/java/com/cloud/runners/CukesRunner.java]: Alexander, could you please delete Tag alev?

**Review by Alexa3462 — APPROVED** [review]
Looks good!

**Review by nurseda916 — APPROVED** [review]

**Pull request merged**